### PR TITLE
Made the stat warning in optimiser point to the wiki

### PR DESCRIPTION
**Pull request by ryan-rushton**
As it needs too much information now.

image

### CODE REVIEW

File: config/i18n.json [hunk 1]
```diff
@@ -560,7 +560,7 @@
     "NumStatMixes_plural": "{{count, number}} stat mixes",
     "OptimizerExplanation": "Select perks, filter and reorder stats, and lock items to narrow down your potential loadouts.",
     "OptimizerExplanationDesktop": "Shift-click a perk to select it. Shift-click an item to exclude it. Drag items to the sidebar to exclude or lock them.",
-    "OptimizerExplanationArmour2Mods": "Stat changes from Armor 2.0 Mods are excluded where possible. To get accurate results, remove all mods with negative stat contributions.",
+    "OptimizerExplanationArmour2Mods": "Stat changes from Armor 2.0 Mods are excluded where possible, for more info please see ",
```
Comment (nev-rtheless) [inline config/i18n.json:563]:
```suggestion
    "OptimizerExplanationArmour2Mods": "Stat changes from Armor 2.0 Mods are excluded where possible, for more info please see {{link}}.",
```
i would prefer letting i18n team decide where link belongs in the sentence

Comment (ryan-rushton) [reply config/i18n.json:563]: Yup I am good with that, tagging @delphiactual for reference.

Comment (ryan-rushton) [reply config/i18n.json:563]: @nev-r and @delphiactual, for components in translations I am assuming I can use this https://react.i18next.com/latest/trans-component? We don't appear to have it in the codebase as of yet.

**Comment by ryan-rushton** [comment]
@nev-r and @delphiactual, I have moved over to using a link in the translation which is just a copy of what we have elsewhere.

**Pull request merged**